### PR TITLE
final rotations added in output,english c-text added,+some minor things

### PR DESCRIPTION
**Pull request by lettyVThrons**

### CODE REVIEW

**Comment by grthor** [comment]
Should I merge it or are you still working on this?

**Comment by grthor** [comment]
The PR has been open for some time now. Should I merge it or is there still work done?

The longer the PR is open, the more likely it is to merge conflicts.

File: org.jcryptool.analysis.fleissner/src/org/jcryptool/analysis/fleissner/UI/TextFoldViewer.java [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+		expandItem.setExpanded(textpresentation.expandedInitially);
+		expandBar.addExpandListener(expandA);
+		expandBar.addControlListener(new ControlAdapter() {
+			@Override
+			public void controlResized(ControlEvent e) {
+				for (Runnable listener : redrawListeners) {
+					parentComposite.getShell().setSize(parentComposite.getShell().getSize().x,
+							parentComposite.getShell().getSize().y + twiddleWindowSize());
+					listener.run();
+					System.out.println("resized");
```
Comment (tassadarius) [inline org.jcryptool.analysis.fleissner/src/org/jcryptool/analysis/fleissner/UI/TextFoldViewer.java:103]: Forgot `System.out.println()` call here

File: org.jcryptool.analysis.fleissner/src/org/jcryptool/analysis/fleissner/logic/MethodApplication.java [hunk 2]
```diff
@@ -322,32 +338,51 @@ public void hillClimbing() {
...
+            String rotation;
+            if (move!=4)
+                rotation = Messages.MethodApplication_output_rotationPos+move;
+            else
+                rotation = Messages.MethodApplication_output_originalPos;
             fg.rotate();
             decryptedText = fg.decryptText(ct.getText());
             value = tv.evaluate(decryptedText);
-                        
+            this.fwAnalysisOutput+=rotation+":\n"+fg+"\n\n"+fg.templateToString(holes)+"\nwith value "+myRound(value)+" and decrypted text:\n\n"+decryptedText+"\n\n";
```
Comment (tassadarius) [inline org.jcryptool.analysis.fleissner/src/org/jcryptool/analysis/fleissner/logic/MethodApplication.java:382]: I think the string  `\nwith value` should be externalized

**Comment by tassadarius** [comment]
Looked through the code, looks good.
2 Points you can take if you want:

- There are many lines of code which just have been commented. In most cases commented lines can be safely deleted without worrying.
- You added the temporary output files of LaTex (*.log, *.aux, *.synctex.gz) under `org.jcryptool.analysis.fleissner/nl/de|en/content/`. These are temporary files and should not be put into git.

Thanks for the contribution and have a nice day.

**Comment by grthor** [comment]
I resolved 2 merge conflicts. 

It might be necessary to pull and merge the changes into your local repo.

**Comment by grthor** [comment]
**Changes from this PR were restructured and revised in PR https://github.com/jcryptool/crypto/pull/314. This PR will therefore be closed and the changes will be added to the crypto repository through PR https://github.com/jcryptool/crypto/pull/314.**

**Pull request closed without merging**